app/mixins/event-wizard.js

```diff
@@ -48,7 +48,7 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
   async saveEventData(propsToSave = []) {
     const { event } = this.model;
     const data = {};
-    const results = await Promise.allSettled(propsToSave.map(property => {
+    const results = await RSVP.allSettled(propsToSave.map(property => {
```

Comment:
Return type of RSVP.allSettled is different from Promise.allSettled. Please fix the code accordingly

Reply:
I tried to get around this, sir. Do review it once. Any specific suggestion or resource to guide me through this?

Reply:
Resolve conflicts and I will test the results